### PR TITLE
Make covers square

### PR DESCRIPTION
**Pull request by deldesir**
Set fixed width and weight to the cover container and adjust the margin of the last book in a row.

### CODE REVIEW

**Review by holta — APPROVED** [review]
I see that `max-width: 180px;` and `height: 225px;` are both shrinking to 150x — which might be a bit small on some tablets?

But if this is tested well enough, Good Enough for Now!

(This will surely evolved over time... as UX requirements become more clear!)

**Review by holta — APPROVED** [review]
> I see that `max-width: 180px;` and `height: 225px;` are both shrinking to 150x — which might be a bit small on some tablets?

@deldesir Can you post a quick explanation to https://github.com/iiab/calibre-web/wiki (or on this ticket is fine too) explaining your thinking here?  So that our choices are more clear when this eventually has to evolve sometime in coming months?

As long as there is some/minimal public documentation explaining this "evolving reasoning" in Plain English... then this PR should definitely be good enough to merge!

**Comment by deldesir** [comment]
The use of square covers is a deliberate decision to improve aesthetics, proving to be a more efficient use of space [in my opinion], whether in the Calibre-web interface or on the screens of various devices.
